conf/messages.cy

```diff
@@ -1198,7 +1198,7 @@ cds.you-cant-use-service-standard-organisation.para2 = Mewngofnodwch fel gweinyd
 cds.sub01.outcome.rejected.subscribe.heading = Mae’r cais am danysgrifiad i {0} ar gyfer {1} wedi bod yn aflwyddiannus
 cds.sub01.outcome.rejected.subscribe.heading-noname = Mae’r cais am danysgrifiad i {0} wedi bod yn aflwyddiannus
 cds.sub01.outcome.rejected.subscribe.title = Mae’r cais am danysgrifiad i {0} wedi bod yn aflwyddiannus
-cds.sub01.outcome.rejected.register.heading = Mae’r cais am gofrestriad {0} ar gyfer {1} wedi bod yn aflwyddiannus
+cds.sub01.outcome.rejected.register.heading = Mae eich cais am rif EORI wedi bod yn aflwyddiannus
```

Comment:
I've been having a think and believe we should also change "cds.sub01.outcome.rejected.register.heading-noname" too. as the other one doesn't have the name in it now anyway so doesn't make sense to leave it as it is